CHANGELOG.md

```diff
@@ -1,6 +1,35 @@
 Changelog
 ================================================================================
 
+2210-2
+--------------------------------------------------------------------------------
+
+### General Notes
+
+⚠️ Before upgrading to this version make sure that there no workflows
+running on older versions of the integration anymore. The new Settings are not
+compatible with older workflow processes.
+
+### Main Changes
+
+* Documentation now clearly states that it is recommended to store the `apiKey`
+in the properties file.
+
+* Configuration has been refactored so that the GlobalLink settings are not 
+automatically published with the pages that they are linked to. 
+
+  **Upgrade Steps:**
+  1. Create new folders for your global- and site-specific Settings at  
+  `/Settings/Options/Settings/Translation Services` and
+  `<SITE_ROOT>/Options/Settings/Translation Service`
+  2. Move the existing GlobalLink settings into the global- and site-specific 
+  folders respectively.
+  3. Remove the links to the GlobalLink settings from the site's pages and publish
+  these pages.
+  4. De-publish the GlobalLink settings and make sure that there are no links to
+  them anymore by checking the system tab. In the future, you don't need to
```

Comment:
"don't need to" -> "should not" (in order not to potentially expose sensitive data).